README.md

```diff
@@ -93,7 +93,112 @@ services:
 
 ### Kubernetes
 
-To Do
+In Kubernetes, you can run the exporter as a sidecar along your "main" container within the same pod.
+
+The following example shows you how to deploy the exporter as a sidecar, accepting logs from the main container via syslog:
+
+```yml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-example
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '4040'
+spec:
+  containers:
+    - name: web
+      image: nginx
+      volumeMounts:
+        - name: nginx-config
+          mountPath: /etc/nginx/nginx.conf
+          subPath: nginx.conf
+          readOnly: true
+    - name: exporter
+      image: valencakarine/nginx-exporter
```

Comment:
@KarineValenca, I think it would be better for the image to be hosted at the labbsr0x docker hub.